tests/testthat/test_kernel.r

```diff
@@ -0,0 +1,6 @@
+context('kernel')
+
+status <- system2('python3', 'test_ir.py', wait=TRUE)
```

Comment:
...or use a series of system commands to install Anaconda ;-). But it may well block network access.